evasquez26/angular4-startup
# S4-ONE-94 Organization details

## Story / Task

see [ONE-94](https://bixlabs.atlassian.net/browse/ONE-94)


## What this PR does?

Adds functionality to see an organization details and courses

### Before 
No organization details page

### After

We can select an organization from the list and see its details

### How to test / reproduce

- Click Schools in the left menu
- See the list of schools. Click any school.
- You should see the school info
- Click courses.
- You should see the school courses.

### Related PRs
N/A

## Migrations

NO


Thank you
